CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+5.0.1 / Work in progress
```

Comment:
if 5.0.0 is still work in progress then why we create 5.0.1? Also, I don't see that the service version has been changed.

Reply:
This pull request should be merged after core 5.0.0 release so the service version would be changed to 5.0.1-SNAPSHOT already.

Reply:
We (likely) won't be merging this before the release (since we are during the code freeze phase)